FindNearby/package.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Commenting here rather than on the large (and human-meaningless 😉) `package-lock.json` and `yarn.lock` files: you want one or the other of these files, i.e. only one package manager: npm or yarn, not both. Things can get tricky if you use two.

We use yarn at the top of the repo, so that's not a bad one to continue on with in this sub-project too—but for the moment I think either is okay. Whichever you choose, just delete the other lock file from this PR.

Reply:
@khawkins jumping in here,  at least in my case (Maybe Rotem as well @rotem-hen-sf  ) I'm using sfdx force:project:create to create the project and it's done with npm - how are you setting up for yarn?

Reply:
Yarn and NPM are both parity-equivalent package managers for Node.js. both work against the same `package.json`, both install packages (in their own way) by creating a `node_modules/` folder worth of dependencies, etc. Which one you use is largely based on personal preferences. They both do things the "Node package" way in their execution. They just use different commands to do so.

`force:project:create` stays silent on your choice of package manager. It creates a `package.json` that either package manager can use, and leaves it up to the consumer to decide.

The way you can typically tell which one a project maintainer has chosen, is by the lock file in the repo. NPM will write a `package-lock.json` as part of its package install, while Yarn will write a `yarn.lock` file. It's basically up to the user to use the package manager that the package maintainer has chosen, implicit through the presence of one or the other of these files.